### PR TITLE
Added pf9-packages name in check

### PR DESCRIPTION
**Pull request by devidask27**
Previously we were just checking for 'pf9-' . Now Full package name is added.

### CODE REVIEW

File: pkg/platform/centos/centos.go [hunk 1]
```diff
@@ -106,13 +106,22 @@ func (c *CentOS) checkDocker() error {
 }
 
 func (c *CentOS) checkExistingInstallation() (bool, error) {
-
-	out, err := c.exec.RunWithStdout("bash", "-c", "yum list installed | { grep -i 'pf9-' || true; }")
-	if err != nil {
-		return false, nil
+	var pf9Packages = []string{"pf9-hostagent", "pf9-comms", "pf9-kube", "pf9-muster"}
```
Comment (nikhilbandi) [inline pkg/platform/centos/centos.go:109]: If we are hardcoding names of pf9 packages, would it make sense to have in the config file?

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
Please look at the minor comment.

File: pkg/platform/centos/centos.go (hunk 1 above)
Comment (devidask27) [reply pkg/platform/centos/centos.go:109]: Do you mean constant file.

**Review by nikhilbandi — APPROVED** [review]
Changes look good.

**Pull request merged**